drivers/i2c/i2c_ll_stm32.c

```diff
@@ -164,17 +157,15 @@ static int i2c_stm32_transfer(struct device *dev, struct i2c_msg *msg,
 		num_msgs--;
 	}
 exit:
-#if defined(CONFIG_I2C_STM32_V1)
-	LL_I2C_Disable(i2c);
```

Comment:
Does polling mode works after removing it?

Reply:
Oups, thanks. Now - yes.